ReactAndroid/src/main/java/com/facebook/react/uimanager/UIViewOperationQueue.java

```diff
@@ -156,16 +156,19 @@ public void execute() {
   private final class ManageChildrenOperation extends ViewOperation {
 
     private final @Nullable int[] mIndicesToRemove;
+    private final @Nullable int[] mTagsToRemove;
     private final @Nullable ViewAtIndex[] mViewsToAdd;
     private final @Nullable int[] mTagsToDelete;
 
     public ManageChildrenOperation(
         int tag,
         @Nullable int[] indicesToRemove,
+        @Nullable int[] tagsToRemove,
         @Nullable ViewAtIndex[] viewsToAdd,
         @Nullable int[] tagsToDelete) {
       super(tag);
       mIndicesToRemove = indicesToRemove;
+      mTagsToRemove = tagsToRemove;
```

Comment:
Should I remove `indicesToRemove` in the end, or we should leave it for any backward compat case?
Also is there any news/plans on reviewing this one? :) Would be awesome to have it merged